src/main/java/friday/Storage.java

```diff
@@ -0,0 +1,41 @@
+package friday;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.File;
+
+/**
+ * Represents a storage system to save and view tasks.
+ */
+public class Storage {
+
+    private File storage;
+    private String filePath;
+    /**
```

Comment:
Should there be no extra lines between the class declaration and the class attribute declarations?

```suggestion
    private File storage;
    private String filePath;
    
    /**
```